Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.ts

```diff
@@ -13,20 +15,30 @@ export class AppComponent {
   weather = 50;
   speed = 0;
   battery = 0;
+  range = 0;
+  speedInput = new FormControl(null, [Validators.required, Validators.min(0), Validators.max(90)]);
+  batteryInput = new FormControl(null, [Validators.required, Validators.min(0), Validators.max(100)]);
+  outputMessage = ""
+  buttonPress = false;
```

Comment:
I know some of the variables (weather, speed, and battery) were initialed with their declaration, but it is preferred to initialize the variables in the constructor.

Something like this:
![image](https://user-images.githubusercontent.com/29678619/139556332-2f49f11d-c7c4-44d1-a140-da70ef25c0bf.png)